### PR TITLE
[BN-1729]

### PR DESCRIPTION
**Pull request by IdaszakDaniel**

### CODE REVIEW

File: docs/TABBEDHEADER.MD [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+  </View>
+)
+
+```
+
+## Custom tab styling
+
+When It comes to tab styling, in TabbedHeader there are seven props:
+
+`tabText` and `tabTextActiveStyle` to pass styles for tab text wheter is inactive or active
```
Comment (rozaliapietrzak) [inline docs/TABBEDHEADER.MD:57]: is it tabText or tabTextStyle? It seems like all other visual props have style/color etc. Does this one not have it?

File: src/StickyParallaxHeader.js [hunk 2]
```diff
@@ -219,13 +219,14 @@ class StickyParallaxHeader extends Component {
...
-    const { foreground, tabsContainerBackgroundColor } = this.props
+    const { foreground, tabsContainerBackgroundColor, backgroundImage } = this.props
 
     return (
       <View
         style={{
           height: backgroundHeight,
-          backgroundColor: tabsContainerBackgroundColor
+          backgroundColor: tabsContainerBackgroundColor,
+          ...( backgroundImage && styles.transparentBackground )
```
Comment (rozaliapietrzak) [inline src/StickyParallaxHeader.js:229]: Does this logic assume that image must always have a background itself. What if someone adds a logo as a background? Should logic not be that by default for image: background color is transparent, however, a useer can override it by passing custom backgroundColor? Just out of curiousity ;-)

File: src/StickyParallaxHeader.js [hunk 3]
```diff
@@ -367,7 +370,7 @@ class StickyParallaxHeader extends Component {
 StickyParallaxHeader.propTypes = {
   background: node,
   backgroundColor: string,
-  backgroundImage: number,
+  backgroundImage: oneOfType([object, number]),
```
Comment (rozaliapietrzak) [inline src/StickyParallaxHeader.js:373]: Also, out of curiousity: why is backgroundImage a number?

File: docs/TABBEDHEADER.MD (hunk 1 above)
Comment (IdaszakDaniel) [reply docs/TABBEDHEADER.MD]: changed prop name

File: src/StickyParallaxHeader.js (hunk 2 above)
Comment (IdaszakDaniel) [reply src/StickyParallaxHeader.js:246]: Currently backgroundImage is under foreground component, that’s why It’s positioned absolutely and with transparent background color. To set custom background color under backgroundImage we need another prop - adding this idea to backlog.

File: src/StickyParallaxHeader.js (hunk 3 above)
Comment (IdaszakDaniel) [reply src/StickyParallaxHeader.js]: changed prop type

**Pull request merged**